front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -275,7 +223,7 @@ export default function Study() {
                 />
               </div>
               <div className="row">
-                <div className="col-xl-9 col-lg-8 col-md-7">
+                <div className="col-xl-12">
```

Comment:
Avoid `-12` without necessity, just remove the two `divs` and we'll be good, thx !